app/helpers/companies_helper.rb

```diff
@@ -0,0 +1,4 @@
+# frozen_string_literal: true
+
+module CompaniesHelper
+end
```

Comment:
I think you don't need that helper for now. I would consider removing this file for now. You can add it later when you really need it :)